### DISCUSSION TITLE
The sensor box changes size when zooming

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#1985] [Q&A]
In trying to set a sensor box to mimic de field of view of my mobile phone camera, I notice that the sensor box apparent size changes as I zoom in or out... This looks quite annoying when you try hard to find the exact apparent field of view... Is it a known bug?

**Comment by gzotti** [comment]
In what way should the sensor box change size? Of course, when you zoom in, the box will appear larger just like the part of the sky enclosed in it. The size of the sensor box depends on sensor size and telescope settings.

**Reply by diegorodriguezperegrin** [reply]
Hehe, yes. I would expect the part of the enclosed sky to change at the same pace as the rectangle, but is not: When zooming in, the rectangle starts to shrink respect to the sky, so stars that were inside at 1st may end outside...

**Reply by axd1967** [reply]
@diegorodriguezperegrin 
could you please add screenshots? example:
![image](https://user-images.githubusercontent.com/3529789/138700275-02004abb-ea45-47c9-b8d0-1953a4427f01.png)
![image](https://user-images.githubusercontent.com/3529789/138700300-9af616ac-786c-457e-a2a1-119624116189.png)

**Comment by gzotti** [comment]
Ar you talking about version 0.21.2 of the desktop version? On what platform? What kind of screen? Display scaling setting? We cannot reproduce what you seem to experience. Maybe show screenshots of what you are doing. How large does this error appear? If this is just a matter of a few percent around the corners in a wide-angle view, also the projection may play a role, as the box in wide-angle views does not follow the distortions of the projection. Switch to perspective projection for the most accurate match.
